src/components/MyKiva/BadgeModalContentJourney.vue

```diff
@@ -143,6 +143,11 @@ const getBadgeStatus = index => {
 	return BADGE_LOCKED;
 };
 
+const getTierName = index => {
+	const contentfulData = badgeWithVisibleTiers.value.contentfulData[index];
+	return `${contentfulData.challengeName} ${contentfulData.levelName}`;
```

Comment:
I think we should guard these to return an empty string if either are undefined...

Reply:
Sure, can do, though if data is undefined, it means we messed up adding contentful data.

Reply:
Both should be defined to have a understandable level name, so I added a fallback to a simple level name.